tools/src/test/resources/1.4/valid-release-notes-1.4.json

```diff
@@ -0,0 +1,194 @@
+      "name": "Stock ticker service",
+      "version": "2020-Q2",
+      "description": "Provides real-time stock information",
+      "endpoints": [
+        "https://partner.org/api/v1/lookup",
+        "https://partner.org/api/v1/stock"
+      ],
+      "authenticated": true,
+      "x-trust-boundary": true,
+      "data": [
+        {
+          "classification": "PII",
+          "flow": "inbound"
+        },
+        {
+          "classification": "PIFI",
+          "flow": "outbound"
+        },
+        {
+          "classification": "pubic",
+          "flow": "bi-directional"
+        },
+        {
+          "classification": "partner-data",
+          "flow": "unknown"
+        }
+      ],
+      "licenses": [
+        {
+          "license": {
+            "name": "Partner license"
+          }
+        }
+      ],
+      "externalReferences": [
+        {
+          "type": "website",
+          "url": "http://partner.org"
+        },
+        {
+          "type": "documentation",
+          "url": "http://api.partner.org/swagger"
+        }
+      ],
+      "releaseNotes": {
+        "type": "major",
+        "title": "My new release",
+        "featuredImage": "https://example.com/featured_image.png",
+        "socialImage": "https://example.com/social_image.png",
+        "description": "The main description of your release",
+        "timestamp": "2021-09-17T00:51:18+00:00",
+        "aliases": [
+          "Project Orion"
+        ],
+        "tags": [
+          "CMS",
+          "SEO",
+          "wysiwyg"
+        ],
+        "resolves": [
+          {
+            "type": "enhancement",
+            "id": "JIRA-17240",
+            "description": "Great new feature that does something",
+            "source": {
+              "name": "Acme Org",
+              "url": "https://issues.example.com/17240"
+            }
+          },
+          {
+            "type": "security",
+            "id": "CVE-2019-9997",
+            "name": "CVE-2019-9997",
+            "description": "Great new feature that does something",
+            "source": {
+              "name": "NVD",
+              "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-9997"
+            },
+            "references": [
+              "http://some/other/site-1",
+              "http://some/other/site-2"
+            ]
+          }
+        ],
+        "notes": [
+          {
+            "locale": "en-US",
+            "text":  {
+              "contentType": "text/html",
+              "encoding": "base64",
+              "content": "PGgxPk15IG5ldyByZWxlYXNlPGgxPgo8cD5SZWxlYXNlIG5vdGVzIGhlcmU8L3A+"
+            }
+          },
+          {
+            "locale": "es",
+            "text":  {
+              "contentType": "text/html",
+              "encoding": "base64",
+              "content": "PGgxPk15IG5ldyByZWxlYXNlPGgxPgo8cD5Ob3RhcyBkZSBsYSB2ZXJzacOzbiBhcXXDrTwvcD4="
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
```

Comment:
Trivial: newline

Reply:
Resolved in efed5c41ca8653629f1795084f0fe24078919eca